source/mir/ndslice/topology.d

```diff
@@ -66,6 +67,7 @@ $(T2 pack     , Returns slice of slices.)
 $(T2 ipack    , Returns slice of slices.)
 $(T2 unpack   , Merges all dimension packs.)
 $(T2 evertPack, Reverses dimension packs.)
+$(T2 byDim    , Convenience function to return a slice that can be iterated by dimension.)
```

Comment:
Please also add a note what actually this function do: transposes dimensions on top and packs them.